### ISSUE TITLE
Reviewer 1 Comment 3

### ISSUE THREAD

**Issue by greenbase** [CIMTT-Kiel/cad-representation-selection#5]
"3. For reproducibility, the paper should include more implementation details: network
depth, hidden sizes, optimizer, learning rate, batch size, number of epochs, regularization,
early stopping criteria, and hardware used. At present, it is difficult to judge whether the
performance differences are due to representation quality or training choices.

Änderungen entsprechend dem Kommentar oben am Paper vornehmen und für Antwort auf Kommentar dokumentieren was geändert worden ist.
